drltrace/drltrace.config

```diff
@@ -0,0 +1,8 @@
+; The syntax of this config file is simple. To add new function, it's enough to specify
+; each argument of the function, function name itself and return type separated by commas.
```

Comment:
As I pointed out in my first comment, a function type with more than one argument has commas inside it.  Some character that cannot appear in a type should be chosen rather than a comma.